docs/nx-cloud/enterprise/dte/azure-dte.md

```diff
@@ -54,8 +54,12 @@ jobs:
             echo "Last successful commit SHA: $LAST_SHA"
             echo "##vso[task.setvariable variable=BASE_SHA]$LAST_SHA"
           fi
+        displayName: 'Get last successful commit SHA'
+        condition: ne(variables['Build.Reason'], 'PullRequest')
+        env:
+          AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
 
-      - script: git branch --track main origin/main
```

Comment:
This should not be changed

Reply:
When you use full fetch depth, main branch will be available.

Reply:
reverted